protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.1.0
+
+* Introduce extension methods `GeneratedMessage.rebuild` and
```

Comment:
Mention what version of protoc_plugin is required.
Perhaps also mention that the sdk constraint has moved

Reply:
Done

Reply:
Done